srcpkgs/qsudo/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'qsudo'
+pkgname=qsudo
+version=2019.11.29
+revision=1
+build_wrksrc="src-qt5"
+build_style=qmake
+hostmakedepends="qt5-host-tools"
+makedepends="qt5-devel"
+depends="qt5 sudo"
+short_desc="Graphical sudo utility from Project Trident"
+maintainer="Ken Moore <ken@project-trident.org>"
+license="BSD-2-Clause"
+homepage="https://github.com/project-trident/qsudo"
+distfiles="https://github.com/project-trident/qsudo/archive/v${version}.tar.gz"
+checksum="0768f4dcc59dadb63b1803f6896b59d78f606224e223d2004ca5cba663a1c17b"
+tags="trident sudo kdesu gksu"
+
+if [ -n "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-tools-devel"
+fi
```

Comment:
please get rid of this and unconditionally add this to hostmakedepends

Reply:
I would expect that adding `qt5-qmake` might be enough. `qt5-devel` is nearly always overkill (except when qmake somehow checks for host headers again instead of target headers)

Reply:
Both changes committed: Removed cross-build section and added qt5-qmake instead of qt5-tools-devel in hostmakedepends.